libr/bin/obj.c

```diff
@@ -101,10 +101,39 @@ R_API RBinObject *r_bin_object_new(RBinFile *binfile, RBinPlugin *plugin, ut64 b
 	return o;
 }
 
+static void r_bin_filter_classes(RBinFile *bf, RList *list) {
```

Comment:
this should be R_API or we should reveme the `r_bin_` prefix.

Reply:
done